src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/spec/DuplicateGerritListenersPreloadedProjectHudsonTestCase.java

```diff
@@ -355,13 +350,16 @@ XmlPage loadConfigXmlViaHttp(TopLevelItem job) throws IOException, SAXException
     /**
      * Checks the size of the listeners collection retrieved by {@link #getGerritEventListeners()}.
      *
-     * @param number the number to assert.
+     * @param extra number of added listeners, other than the default.
      */
-    void assertNrOfEventListeners(int number) {
+    void assertNrOfEventListeners(int extra) {
         Collection<GerritEventListener> gerritEventListeners = getGerritEventListeners();
-        // DependencyQueueTaskDispatcher adds 1 listener
-        // ReplicationQueueTaskDispatcher adds 1 listener
-        assertEquals(number, gerritEventListeners.size());
+        int nbrOfListeners = extra;
+        nbrOfListeners++; // EventListener adds 1 listener
+        nbrOfListeners++; // DependencyQueueTaskDispatcher adds 1 listener
+        nbrOfListeners++; // ReplicationQueueTaskDispatcher adds 1 listener
+        nbrOfListeners++; // GerritProjectListUpdater adds 1 listener
+        assertEquals(nbrOfListeners, gerritEventListeners.size());
```

Comment:
That means good :) Its alot more cleaner and easier to read than what the test did previously.